DbgPack/__init__.py

```diff
@@ -1,7 +1,11 @@
-from .asset import Asset
+from .asset1 import Asset1
+from .asset2 import Asset2
+from .loose_asset import LooseAsset
+from .asset_manager import AssetManager
 from .pack1 import Pack1
 from .pack2 import Pack2
+from .loose_pack import LoosePack
 from .struct_reader import BinaryStructReader
-from .asset_manager import AssetManager
 
-__all__ = ['Asset', 'AssetManager', 'Pack1', 'Pack2', 'BinaryStructReader']
+__all__ = ['Asset1', 'Asset2', 'LooseAsset', 'Pack1', 'Pack2', 'LoosePack',
+           'AssetManager', 'BinaryStructReader']
```

Comment:
```suggestion
           'AssetManager']
```
Same reason: no need to expose a utility class.